Dockerfile

```diff
@@ -32,6 +32,14 @@ RUN cd /opt/ \
   && wget https://archive.apache.org/dist/spark/spark-2.3.0/spark-2.3.0-bin-hadoop2.6.tgz \
   && tar -xvf ./spark-2.3.0-bin-hadoop2.6.tgz \
   && rm -rf /opt/spark-2.3.0-bin-hadoop2.6.tgz
+RUN cd /opt/ \
```

Comment:
May be we should remove `python3-dev` this from the above then?
And if we anyway install from sources - why not use the latest and greatest - 3.7.4, or it may break smth?

Reply:
i don't think it's gonna break something, i'll add 3.7